src/python/interop/reddit/adapter.py

```diff
@@ -0,0 +1,68 @@
+from grongier.pex import InboundAdapter
```

Comment:
do we need to add interoperability to the basic embedded python tempate at all? we have the dedicated interoperability python template for it, right?

Reply:
it can be an all in one demo, I can remove it, if it confuse ppl